source/reflect.h

```diff
@@ -3626,19 +3627,20 @@ parse_context_branch_reset_state::parse_context_branch_reset_state(){}
         return r; 
     }
 
-#line 1966 "reflect.h2"
+#line 1967 "reflect.h2"
     [[nodiscard]] auto parse_context::has_next() const& -> bool{return cpp2::impl::cmp_less(pos,regex.size()); }
 
-#line 1968 "reflect.h2"
+#line 1969 "reflect.h2"
     [[nodiscard]] auto parse_context::grab_until_impl(cpp2::impl::in<std::string> e, cpp2::impl::out<std::string> r, cpp2::impl::in<bool> any) & -> bool
     {
-        auto end {pos}; 
+        auto end {[&]() mutable -> auto{
```

Comment:
The instantly evaluated lambda seems a bit of a mouthful here. The following should work:
```suggestion
        const auto end = any ? regex.find_first_of(e, pos) : regex.find(e, pos);
```

Reply:
I had a similar thought.

Reply:
the stuff in reflect.h is generated code, please provide alternative syntax for reflect.h2 and I will update it!